internal/lib/container_server.go

```diff
@@ -468,7 +503,7 @@ func (c *ContainerServer) ContainerStateFromDisk(ctr *oci.Container) error {
 // ContainerStateToDisk writes the container's state information to a JSON file
 // on disk
 func (c *ContainerServer) ContainerStateToDisk(ctr *oci.Container) error {
-	if ctr == nil {
+	if ctr.Spoofed() {
```

Comment:
Do we need a skip in `ContainerStateFromDisk` as well?

Reply:
we can for posterity, but the only time it's called is when restoring, in a block where we don't restore if the container used to be spoofed.